proto/xudp2p.proto

```diff
@@ -26,6 +26,7 @@ message NodeState {
     string raiden_address = 5;
     map<string, string> lnd_pub_keys = 6;
     map<string, string> token_identifiers = 7;
+    map<string, string> lnd_uris = 8;
```

Comment:
As I mentioned you should take a look at using `map<string, ListValue>` instead of a comma delimited string here. It might be a cleaner approach.